openproblems/tasks/_batch_integration/batch_integration_feature/methods/scalex.py

```diff
@@ -0,0 +1,15 @@
+from ...batch_integration_graph.methods.scalex import _scalex
+from ...batch_integration_graph.methods.scalex import _scalex_method
+from typing import Optional
+
+
+@_scalex_method(method_name="SCALEX")
+def scalex_full(adata, test: bool = False, max_iteration: Optional[int] = None):
+    return _scalex(
+        adata,
+        test=test,
+        max_iteration=max_iteration,
+        compute_neighbors=False,
+        compute_features=True,
+        n_top_features=0,
+    )
```

Comment:
why no `scalex_hvg` here?

Reply:
Maybe I'm misunderstanding the feature task -- if scalex filters down to HVGs, isn't that then the wrong shape output?

Reply:
I think there's only 1 feature-specific metric and it's HVG overlap. We subset to the same genes before and after to assess the performance here, so it doesn't matter. We can deal with different numbers of features. 

I guess the only real question is: is the HVG-selection SCALEX-specific? If so, we'd also be evaluating this as part of the method, meaning it's no longer about comparing HVG selection in the same way across methods (which might be fine).

Reply:
Gotcha. I'm inclined to say that if a method specifically offers HVG selection (in fact it's the default SCALEX behaviour) we should use their built-in HVG selection, because that's how it will be used in the real world.

Reply:
I would probably agree... just can't use this task to compare HVG selection as input to batch integration methods then anymore.

Reply:
Gotcha. TBH I'd consider that its own task but I see your point (: Happy for me to merge?